lib/aws/rails/action_mailbox/engine.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
-require 'action_mailbox/engine'
-
 module Aws
   module Rails
     module ActionMailbox
       # @api private
       class Engine < ::Rails::Engine
-        config.action_mailbox.ses = ActiveSupport::OrderedOptions.new
-        config.action_mailbox.ses.s3_client_options ||= {}
+        if config.respond_to?(:action_mailbox)
```

Comment:
Should we also check that ActionMailbox is loaded? (ie, check if the module `ActionMailbox` is defined?)

Reply:
ActionMailbox can be loaded (i.e. `require 'action_mailbox'`) but this can still fail, it really needs `action_mailbox/engine`.